ddtrace/tracer.py

```diff
@@ -933,8 +933,7 @@ def execute():
         """
 
         def wrap_decorator(f: AnyCallable) -> AnyCallable:
-            # FIXME[matt] include the class name for methods.
-            span_name = name if name else "%s.%s" % (f.__module__, f.__name__)
+            span_name = name if name else "%s.%s" % (f.__module__, f.__qualname__)
```

Comment:
Thanks for the contribution 😄 

Is the motivation for this change just to address the comment or would you like to have this default behavior? 

This comment was added years ago when ddtrace was still in a beta state. Updating the default names of spans could break monitors, dashboards, and metrics for many existing datadog customers. Instead of addressing the comment I would prefer to just remove it.

If this feature has the potential to significantly improve your experience with ddtrace then we can hide this change behind an environment variable and make it opt in.

```suggestion
            span_name = name if name else "%s.%s" % (f.__module__, f.__name__)
```